core/src/com/unciv/ui/trade/DiplomacyScreen.kt

```diff
@@ -223,6 +231,52 @@ class DiplomacyScreen(val viewingCiv:CivilizationInfo):CameraStageBaseScreen() {
         return diplomacyTable
     }
 
+    /** Given a list of city-state bonuses, returns the list with updated values for Siam-like uniques */
+    private fun getIncreasedBonuses(bonuses: List<String>): List<String> {
+        val newBonuses = ArrayList<String>()
+        println(bonuses)
```

Comment:
This looks like debug printlns :) (also one at the bottom of this function)

Reply:
Not without context in the output _and_ an if(const val)